front/screens/epdsSurvey/epdsResult.component.tsx

```diff
@@ -9,31 +9,34 @@ import { EpdsSurveyUtils } from "../../utils";
 import IconeResultatBien from "../../assets/images/icone_resultats_bien.svg";
 import IconeResultatMoyen from "../../assets/images/icone_resultats_moyen.svg";
 import IconeResultatPasBien from "../../assets/images/icone_resultats_pasbien.svg";
+import { EpdsIconResult, EpdsIconResultEnum } from "../../type";
 
 interface Props {
   result: number;
 }
 
 const EpdsResult: React.FC<Props> = ({ result }) => {
-  // result = 5;
   const labelsResultats = Labels.epdsSurvey.resultats;
   const {
     stateOfMind,
     label,
     colorStyle,
-    icone
+    icon
   } = EpdsSurveyUtils.getResultLabelAndStyle(result);
 
-  let resultIcone;
-  if (icone === "BIEN") resultIcone = { image: <IconeResultatBien /> };
-  else if (icone === "MOYEN") resultIcone = { image: <IconeResultatMoyen /> };
-  else resultIcone = { image: <IconeResultatPasBien /> };
+  const getIcon = (icone: EpdsIconResultEnum) => {
+    let iconsMap = new Map<EpdsIconResultEnum, JSX.Element >(); 
```

Comment:
Désolé, j'ai parlé d'une map mais je pensais à un objet :D Bon après les deux implems marchent (les données sont séparées de la logique) donc on garde la version que tu trouves la plus claire. 

Pour le type des icônes, tant mieux si `JSX.Element` ne produit par d'erreur, mais en cas de problème tu peux utiliser `ReactNode`. La différence est expliquée ici : https://stackoverflow.com/questions/58123398/when-to-use-jsx-element-vs-reactnode-vs-reactelement

Enfin, il y a un espace entre `JSX.Element` et `>` dont il faut peut-être lancer `yarn lint-fix`